pkg/generators/generator_spec_processor_test.go

```diff
@@ -0,0 +1,57 @@
+package generators
```

Comment:
It seems you are just using exported symbols having the test validation from the consumer perspective (blackbox test). This is really good type of test to make sure you are exporting just the necessary symbols. In this case you can define the package name with the `_test` suffix (`generators_test`) which is a Go builtin functionality. More on this here:
https://stackoverflow.com/questions/19998250/proper-package-naming-for-testing-with-the-go-language

Reply:
Ah nice, that's precisely what I searched for and somehow missed.